cmd/podman/common/build.go

```diff
@@ -65,11 +65,11 @@ func DefineBuildFlags(cmd *cobra.Command, buildOpts *BuildFlagsWrapper) {
 
 	// --pull flag
 	flag := budFlags.Lookup("pull")
-	if err := flag.Value.Set("true"); err != nil {
+	flag.DefValue = "missing"
+	if err := flag.Value.Set("missing"); err != nil {
 		logrus.Errorf("Unable to set --pull to true: %v", err)
```

Comment:
This is probably an impossible code path, but shouldn't it be `s/true/missing/`?

Reply:
Argh, good catch.  It's probably impossible but I let you decide to merge or let me fix it :^)